### PR TITLE
I've implemented Login/logout features

### LINKED ISSUE #125: Authorized Shell view for Mobile

### LINKED ISSUE #127: Login view on Mobile

### PR DESCRIPTION
**Pull request by ZhakalenDk**
This PR closes #125 and closes #127, however, som features haven't been tested enough because there's an issue with our services.
THere's also some refresh token problems, although I suspect these issues are related to the way we push auth data to the API.

I believe @Sjimooon is activly trying to figure out what's going on and implement safe guards on the API so the changes in this PR will do for now.

### CODE REVIEW

File: PubHub/PubHub.Common/ErrorsCodeConstants.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿namespace PubHub.Common
+{
+    public static class ErrorsCodeConstants
+    {
+        public const string NO_CONNECTION = "0x00001";
+        public const string INVALID_TOKEN = "0x00002";
```
Comment (Sjimooon) [inline PubHub/PubHub.Common/ErrorsCodeConstants.cs:6]: Just curious. What are thse used for and why in HEX format? 🙂

**Review by jeongoks — APPROVED** [review]
Looks good. I hope everything will work with the changes we make. 🤞

**Review by Sjimooon — APPROVED** [review]
Looks promising, and yes I'm just about done with the JWT token stuff for the API. If things continue to go as planned, a PR will be up later today that will make the JWT authentication actually functional.

File: PubHub/PubHub.Common/ErrorsCodeConstants.cs (hunk 1 above)
Comment (ZhakalenDk) [reply PubHub/PubHub.Common/ErrorsCodeConstants.cs:6]: Those are error codes that are displayed for the end user (:
We don't want to publish debug data when an errors occurs, so having a code that is simply for us to read yet complex enough to support large sets of return codes will aid us in tracking down issues.

**Pull request merged**